### PR TITLE
Describe registry use and conformance

### PR DESCRIPTION
**Pull request by aphillips**
Addresses my comments in #561

### CODE REVIEW

File: spec/registry.md [hunk 1]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
   - Display/edit known message metadata.
   - Restrict input in GUI by providing a dropdown with all viable option values.
 
+## Conformance and Use
+
+_This section is normative._
+
+To be conformant with MessageFormat 2.0, an implementation MUST implement
+all of the _selectors_ and _functions_ described in the default registry,
```
Comment (eemeli) [inline spec/registry.md:33]: Should reword slightly, as _functions_ includes selectors.

```suggestion
To be conformant with MessageFormat 2.0, an implementation MUST implement
all of the formatting and selection _functions_ described in the default registry,
```

File: spec/registry.md [hunk 2]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
...
+described by the default registry.
+
+Implementations are not required to provide a registry nor to read or interpret
+a copy of this registry in order to be conformant.
+
+The MessageFormat 2.0 Registry was created to describe the core set of _selectors_
+and _functions_, including _operands_, _options_, _option_ values, as well
+as expected outputs.
+The descriptions in this registry are intended to describe the specific required
+signatures for _selectors_ and _functions_ that all implementations of MessageFormat 2.0
```
Comment (eemeli) [inline spec/registry.md:44]:
```suggestion
signatures for _functions_ that all implementations of MessageFormat 2.0
```

File: spec/registry.md [hunk 3]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
...
+all of the _selectors_ and _functions_ described in the default registry,
+including all of the _options_ and _option_ values, _operands_ and outputs
+described by the default registry.
+
+Implementations are not required to provide a registry nor to read or interpret
+a copy of this registry in order to be conformant.
+
+The MessageFormat 2.0 Registry was created to describe the core set of _selectors_
+and _functions_, including _operands_, _options_, _option_ values, as well
+as expected outputs.
```
Comment (eemeli) [inline spec/registry.md:42]: Actually, the registry does not (at least currently) define expected outputs.

```suggestion
The MessageFormat 2.0 Registry was created to describe
the core set of formatting and selection _functions_,
including _operands_, _options_, and _option_ values.
```

File: spec/registry.md [hunk 4]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
...
+The descriptions in this registry are intended to describe the specific required
+signatures for _selectors_ and _functions_ that all implementations of MessageFormat 2.0
+MUST implement.
+The goal is to ensure message interoperability between implementations,
+regardless of programming language or runtime environment.
```
Comment (eemeli) [inline spec/registry.md:47]: I don't think we can _ensure_ interop, but we can at least try.

```suggestion
The goal is to enable message interoperability between implementations,
regardless of programming language or runtime environment.
```

File: spec/registry.md [hunk 5]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
...
+This ensures that developers do not have to relearn core MessageFormat syntax
+when moving between platforms
```
Comment (eemeli) [inline spec/registry.md:49]:
```suggestion
This ensures that developers do not have to relearn core MessageFormat syntax
and functionality when moving between platforms
```

File: spec/registry.md [hunk 6]
```diff
@@ -25,8 +25,45 @@ in order to support the following goals and use-cases:
...
+and that translators do not need to know about the runtime environment for most
+selection or formatting operations.
+
+The registry provides a machine-readable description of _selectors_ and _functions_
```
Comment (eemeli) [inline spec/registry.md:53]:
```suggestion
The registry provides a machine-readable description of _functions_
```

**Review by eemeli — CHANGES_REQUESTED** [review]

**Review by eemeli — APPROVED** [review]

**Review by mihnita — APPROVED** [review]

**Pull request merged**